spec/factories/target_matchers.rb

```diff
@@ -0,0 +1,13 @@
+# == Schema Information
```

Comment:
Since you dont have a target_matchers model/table, this factory doesn't seem to make sense